src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml

```diff
@@ -35,10 +35,16 @@
                 </div>
             </div>
 
+            <h2 class="govuk-heading-m">What happens next</h2>
+
+            <p class="govuk-body"> We'll send the apprentice's certificate to the address you entered. You can <a class="govuk-link" asp-controller="CertificateHistory" asp-action="Index">check the status of the certificate</a> in the 'Completed assessments' section of your account.</p>
```

Comment:
Mostly this page uses a localizer to supply the text - I think it should either continue to mostly do so - for the new text or not do that at all and convert back to inline text completely.

My preference would be to convert it to inline text as mostly the rest of the application does not use localizers - as EFSA is only for England...

Reply:
As per @ChrisJWoodcock  recommendation, remove Localizer for this View.

Reply:
@shomavg - you haven't done this yet